### PR TITLE
Compatibility

### LINKED ISSUE #15: Check WP 6.4 compatibility

### PR DESCRIPTION
**Pull request by javiercasares**
- Prepared for v1.1.0
- WordPress 6.4, checked
- PHPCS + WPCS 3.0.1, checked
- PHP 5.6 -> 8.3, checked

### CODE REVIEW

**Comment by javiercasares** [comment]
@jamieblomerus do you want to check it before launching the new version?

**Comment by jamieblomerus** [comment]
Sure, I am currently in bed trying to sleep. But I will check it tomorrow.

**Comment by jamieblomerus** [comment]
Could you assign me so I don't forget?

File: readme.txt [hunk 1]
```diff
@@ -30,8 +30,54 @@ Extract the contents of the ZIP and upload the contents to the `/wp-content/plug
...
+* [WordPress APIs Security](https://developer.wordpress.org/apis/security/)
+* [WordPress Coding Standards](https://github.com/WordPress/WordPress-Coding-Standards)
+
+== Privacy ==
+
+* This plugin does not collect any information about your site, your identity, the plugins, themes or content the site has.
+
+== Vulnerabilities ==
+
+* No vulnerabilities have been published up to version 2.2.1.
```
Comment (jamieblomerus) [inline readme.txt]: This line says that no vulnerabilities have been published up to version 2.2.1, which does not exist yet.

I would recommend changing this to:
```suggestion
* No vulnerabilities have been published up to version 1.1.0.
```

File: readme.txt [hunk 2]
```diff
@@ -30,8 +30,54 @@ Extract the contents of the ZIP and upload the contents to the `/wp-content/plug
 
 Data is coming from [The Green Web Foundation](https://www.thegreenwebfoundation.org/) API.
 
+== Security ==
+
+For security or privacy concerns, please contact us at [wordpress @ wpsysadmin . com].
```
Comment (jamieblomerus) [inline readme.txt]: In the Plugin Directory, the content from readme.txt files is processed as Markdown, but it doesn't automatically convert email addresses into clickable mailto links unless we format them that way. Additionally, broken-up email addresses like [wordpress @ wpsysadmin . com] may confuse users and are not clickable.

Hence I would recommend changing this to:
```suggestion
For security or privacy concerns, please contact us at wordpress@wpsysadmin.com.
```
or
```suggestion
For security or privacy concerns, please contact us at [wordpress@wpsysadmin.com](mailto:wordpress@wpsysadmin.com).
```

I would also like to add that email addresses not being formatted as such badly affects the accessibility for disabled users.

File: readme.txt [hunk 3]
```diff
@@ -30,8 +30,54 @@ Extract the contents of the ZIP and upload the contents to the `/wp-content/plug
...
+* [WordPress Coding Standards](https://github.com/WordPress/WordPress-Coding-Standards)
+
+== Privacy ==
+
+* This plugin does not collect any information about your site, your identity, the plugins, themes or content the site has.
+
+== Vulnerabilities ==
+
+* No vulnerabilities have been published up to version 2.2.1.
+
```
Comment (jamieblomerus) [inline readme.txt:51]: Perhaps we should include a way for people to report security vulnerabilities.

How about this?
```suggestion
Found a security vulnerability? Please report it to us privately at the [WPsustainable GitHub repository](https://github.com/javiercasares/WPsustainable/security/advisories/new).
```

**Review by jamieblomerus — CHANGES_REQUESTED** [review]
@javiercasares I found some things in the `readme.txt` that need fixing. I have included comments that describe each issue and suggestions on how to fix them.

Except for these things, everything looks good.

File: readme.txt (hunk 3 above)
Comment (javiercasares) [reply readme.txt:51]: I like this idea...

**Comment by javiercasares** [comment]
I think now is ready.

I'll try to create the package in a few hours.

**Review by jamieblomerus — APPROVED** [review]
💯 Looking good! Ready for release.

**Pull request merged**